apps/sms/js/thread_ui.js

```diff
@@ -2100,14 +2100,11 @@ var ThreadUI = {
           {
             l10nId: 'delete',
             method: function deleteMessage(messageId) {
-              Utils.confirm(
-                'deleteMessage-confirmation', null,
-                { text: 'delete', className: 'danger' }
-              ).then(() => {
```

Comment:
actually I'm not sure we should remove the confirmation; I think we should wait for the undo mechanism to be implemented :)